distribution/docs/src/main/resources/content/_managing/_installing/installing-from-admin-console.adoc

```diff
@@ -5,7 +5,7 @@
 :project: ${branding}
 :order: 03
 
-===== Completing Installation from the ${admin-console}
+=== Completing Installation from the ${admin-console}
```

Comment:
These sections still appear off to me:
![image](https://user-images.githubusercontent.com/8041246/52308920-d98a6800-295b-11e9-81b7-84205b92ea12.png)

I would expect:
```
- 6.3.3. Completing Installation
    6.3.4. Completing Installation from the Admin Console
    + 6.3.5. Completing Installation from the Command Console
```

Reply:
new version: 

![toc](https://user-images.githubusercontent.com/10619999/52309900-c2994500-295e-11e9-842a-cf18ccd0af42.png)